### PR TITLE
docs: Capturing that non-exclusive CPUs are not accounted in NRT objects

### PR DESCRIPTION
**Pull request by swatisehgal**
Signed-off-by: Swati Sehgal <swsehgal@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ The operator provides minimal support to deploy [secondary schedulers](https://g
 
 Please check the [issues section](https://github.com/openshift-kni/numaresources-operator/issues) for the known issues and limitations of the NUMA resources operator.
 
+## Addtional noteworthy information
+
+NRT objects only take into consideration exclusively allocated CPUs while accounting. In order for a pod to be allocated exclusive CPUs, it HAS to belong to Guaranteed QoS class (request=limit) and request has to be integral. Therefore, CPUs in the shared pool because of pods belonging to best effort/burstable QoS or guaranteed pod with non-integral CPU request would not be accounted for in the NRT objects. Please refer to CPU Manager docs [here](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy) for more detail on this.
+
+In addtion to this, PodResource API is used to extract the resource information from Kubelet for resource accounting. CPUs exposed by the List endpoint of Podresource API correspond to exclusive CPUs allocated to a particular container. CPUs that belong to the shared pool are therefore not exposed by this API.
```
Comment (shajmakh) [inline README.md:15]: typo: add**i**tion

**Comment by shajmakh** [comment]
looks good! thanks for the update!
/lgtm

**Comment by shajmakh** [comment]
/hold 
feel free to unhold when the typo is fixed

**Comment by swatisehgal** [comment]
/unhold

@shajmakh I fixed the typo, can you please take another quick look and add an LGTM if all loogs good.
Thanks

**Comment by shajmakh** [comment]
Thanks! 
/lgtm

**Review by marioferh — COMMENTED** [review]
/lgtm

**Pull request merged**